virttest/qemu_devices/qdevices.py

```diff
@@ -1364,6 +1367,11 @@ def get_qmp_args(self):
             args["backend"]["data"] = {"device": self.get_param("path")}
             return args
 
+        if self.params.get("backend") in "ringbuf":
+            args["backend"]["data"] = \
+                {"size": self.get_param("ringbuf_write_size")}
```

Comment:
Better avoid `\`,
```
args["backend"]["data"] = {
    "size": self.get_param("ringbuf_write_size")}
```

Reply:
> Better avoid `\`,
> 
> ```
> args["backend"]["data"] = {
>     "size": self.get_param("ringbuf_write_size")}
> ```

Updated , please help to review, thanks.